src/player.cpp

```diff
@@ -828,6 +828,13 @@ void player::process_turn()
             add_msg_if_player( m_info, _( "You learned a new style." ) );
         }
     }
+    // toggle facing direction for sdl flip
+    if ( inp_mngr.get_previously_pressed_key() == 260 ) {
```

Comment:
Use the name for the input value as defined in `input.h` since they're known: `KEY_LEFT` and `KEY_RIGHT`
Additionally, because the previous input key conditions are mutually exclusive could change from
```
if( condition1 ) {
...
}
if( condition2 ) {
...
}
```
to
```
if( condition1 ) {
...
} else if( condition2 ) {
...
}
```

Reply:
this one wouldn't work somehow.
if i use KEY_RIGHT and KEY_LEFT in player.cpp the player sprite won't flip.
if i use` inp_mngr.get_previously_pressed_key() == 260 ` it work as before, i don't understand why. so i did all the rest bust this.

Reply:
ah i think i know, it's because key_left and key_right do not stay true, while get_previously_pressed_key() does, is correct ?

Reply:
ok like this:
get_previously_pressed_key() == KEY_RIGHT
it works fine.